source/_components/climate.markdown

```diff
@@ -20,6 +20,39 @@ climate:
   platform: demo
 ```
 
+## {% linkable_title Properties %}
+
+Every climate device has a set of properties that control the operation of the climate device. These properties can be set via the frontend or by calling services and are reported in the attributes of the device's state. Note that not all platforms implement all the properties.
+
+We explain each of the properties and the possible choices for them shortly:
+
+* The **action** property specifies the overall operational mode that the device is currently in. Choices are:
+ * Cooling
+ * Heating
+ * Auto (Cool or heat, depending on what's necessary to achieve the target temperature)
+ * Off
+* The **target_temperature** property selects the desired temperature for the device to achieve. The device will take measures to do so depending on what `action` is chosen. While most devices allow only to specify a `target_temperature` and will start heating / cooling when the temperature differs too far from that temperature, some devices allow finer control by specifying an  acceptable temperature corridor:
+  * The **target_temp_high** property selects an upper limit on the acceptable temperature. The device will start cooling (if `cooling` or `auto` are selected as `action`) if that temperature is exceeded.
+  * The **target_temp_low** property selects a lower limit on the acceptable temperature. The device will start heating (if `heating` or `auto` are selected as `action`) if that temperature is exceeded.
+* The **target_humidity** property selects the target humidity. If this is set and the device is able to dry / humidify the air, it will do so if the actual humidity differs from this value.
+* The **control** property specifies how the device is currently being controlled. Many devices allow to chose between an automated control (following a given preset, see below) or a manual control. *Note*: This does *not* influence whether the device is controlled by Home Assistant. The device is always controlled by Home Assistant, even if in `manual` mode. This property controls only the setting on the actual device. Choices are:
+ * Preset (device follows the device-specific preset, see below)
+ * Manual (the device itself is only controlled manually / via Home Assistant)
```

Comment:
Good question. Personally I would think that as soon as e.g. `set_temperature` is called, the `control` property is changed to `manual`, i.e. presets are inactive from this point on. If the user calls `set_temperature`, he obviously wants the temperature not to be whatever the current preset says.

Reply:
Thinking about it, that's nonsense. Setting a temporary temperature until the next scheduled temperature change happens makes a lot of sense.

Reply:
I've seen this feature on Tado, where you can have some standard settings and the ability to diverge from those manually based on a condition (until manually moved back or until next automatic change comes into effect). 

This would be a good feature to add, to allow comparison between presets and automation (in case the component allows it) and manual changes done to temperature through the UI.